packages/core/tsconfig.json

```diff
@@ -8,7 +8,8 @@
         "noPropertyAccessFromIndexSignature": true,
         "noImplicitReturns": true,
         "noFallthroughCasesInSwitch": true,
-        "exactOptionalPropertyTypes": true,
+        //"exactOptionalPropertyTypes": true,
```

Comment:
Is this temporarily commented out or can it be removed?

Reply:
Removed, hard to use mad-components with that setting enabled, when mad-components does not have it enabled